tests/playbooks/kind_crud.yml

```diff
@@ -597,12 +587,11 @@
                       memory: 512M
                   devices:
                     disks:
-                    - volumeName: myvolume
-                      name: mydisk
+                    - name: myvolume
                       disk:
                         bus: virtio
                     - name: cloudinitdisk
-                      volumeName: cloudinitvolume
+name: cloudinitvolume
```

Comment:
Actually this would break the yaml. Please align it correctly.

Reply:
Thanks for noticing. Done.